Bottom: 0f29fc79a2e44b45051dc6d32b885f356ac262aa
Top:    7f355c2c4b9a69f80c172a756e9357d22ad478bf
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-24 16:17:57 -0700

Refresh of rs-iomap

---

diff --git a/src/rsocket.c b/src/rsocket.c
index c534c61..afcc5b1 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1453,7 +1453,7 @@ static int rs_send_iomaps(struct rsocket *rs, int flags)
 			break;
 	}
 
-	rs->iomap_pending = dlist_empty(&rs->iomap_queue);
+	rs->iomap_pending = !dlist_empty(&rs->iomap_queue);
 	fastlock_release(&rs->iomap_lock);
 	return ret;
 }
